lib/reservation_generator_helper.rb

```diff
@@ -0,0 +1,96 @@
+module ReservationGeneratorHelper
+  require 'ffaker'
```

Comment:
Do we need to `require` this manually? We use it in `BlackoutGenerator` and don't explicitly call `require`, so I'm pretty sure it pulls it in from the call in `seeds.rb`.